src/Controller/EnMarche/MembershipController.php

```diff
@@ -107,6 +145,24 @@ public function joinAction(Request $request, EntityManagerInterface $manager): R
         ]);
     }
 
+    /**
+     * Send an email to validate email
+     *
+     * @Route("/valider-email", name="app_membership_email_validation")
+     * @Security("is_granted('ROLE_USER')")
+     */
+    public function sendEmailValidationAction(Request $request): Response
```

Comment:
why this action?

Reply:
.... totaly forgot to delete it after the revert :'(